blazor/file-manager/pagination.md

```diff
@@ -0,0 +1,42 @@
+﻿---
+layout: post
+title: Pagination in Blazor FileManager Component | Syncfusion
+description: Checkout and learn here all about Pagination in the Syncfusion Blazor FileManager component and more.
+platform: Blazor
+control: File Manager
+documentation: ug
+---
+
+# Pagination in Blazor FileManager Component
+
+Pagination in the Blazor File Manager component optimizes the performance and user experience by loading only a specified number of files and folders per page. This feature significantly improves performance when managing large datasets, reducing load times and enhancing interaction.
+
+## Enabling Pagination
+
+To enable pagination in the File Manager, set the [AllowPaging](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.SfFileManager-1.html#Syncfusion_Blazor_FileManager_SfFileManager_1_AllowPaging) property to `true`. This enables pagination for both Details View and Large Icons View, ensuring efficient navigation and file management across various display modes.
```

Comment:
details view and large icons view do not use capital words in between if we are not mentioning that as API

Reply:
modified the changes and updated the PR.